neon/pgstatementsextensiondoc.md

```
@@ -0,0 +1,286 @@
+
+### Query the pg_stat_statements view
+
+The main interface is the `pg_stat_statements` view, which contains one row per distinct database query, showing various statistics.
+
+```sql
+SELECT * FROM pg_stat_statements LIMIT 10;
+```
+
+The view contains details like those shown below:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)